config/manifests/bases/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -27,12 +27,13 @@ spec:
     or by specifying existing instances through a [ConfigMap](https://docs.openshift.com/container-platform/latest/windows_containers/configuring-byoh-windows-instance.html)
```

Comment:
@selansen , You may need to run `make bundle` to reflect the changes in the bundle.

Reply:
why do I need to run make bundle? Looked into previous changes in the same file, haven't noticed anyone ran make bundle for this file change. Am I missing anything?

Reply:
I am not creating any bundle image.

Reply:
> why do I need to run make bundle? Looked into previous changes in the same file, haven't noticed anyone ran make bundle for this file change. Am I missing anything?

`make bundle` will apply the changes in the bundle CSV, [here](https://github.com/openshift/windows-machine-config-operator/blob/master/bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml#L36). Otherwise won't take effect.

For references, you can take a look at https://github.com/openshift/windows-machine-config-operator/pull/780.

Reply:
> This will be done as part of release prep, right?

Yes, and running `make bundle` now will apply the changes in the [bundle CSV](https://github.com/openshift/windows-machine-config-operator/blob/master/bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml#L36).